src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -267,6 +279,25 @@ Please note that the `<index>` denotes the number/index (in a form of an
 integer) of the echo not the echo time value which needs to be stored in the
 field EchoTime of the separate JSON file.
 
+Complex-valued data MUST be split into one file for each data type.
+For BOLD data, there are separate suffices for magnitude (`_bold`) and phase
+(`_phase`) data.
```

Comment:
echoing on above I would have preferred `1.x` was "forward looking" and allowed `_part-<mag|phase>` for `_bold` while announcing `_phase` deprecated.  Similar to how such deprecations are done in software.  That would ease later migration to BIDS 2.0.  It would be only a slight burden for bids-apps to support both layouts IMHO, but would also simplify their support for BIDS 2.0.

Reply:
I agree with @yarikoptic. I would suggest that we say `_part-<mag|phase>`, and then say "For backwards compatibility, `_phase.nii.gz` is considered equivalent to `_part-phase_bold.nii.gz`."

Reply:
I didn't realize that was possible, but I certainly love the idea. By the way, for posterity's sake, I have an open issue in `bids-2-devel` about eliminating `_phase` (and other suffixes that could be better handled with this entity): https://github.com/bids-standard/bids-2-devel/issues/3.

Reply:
Should be good now (although the wording could use a review).